mobile/src/bundles/common/components/overlay/overlay.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {
+    ActivityIndicator,
+    View,
+} from '~/bundles/common/components/components';
+
+import { Color } from '../../enums/enums';
+import { globalStyles } from '../../styles/global-styles';
+import { styles } from './styles';
+
+type OverlayProperties = {
+    isActive: boolean;
+    color?: string;
+};
+const Overlay: React.FC<OverlayProperties> = ({
+    isActive,
+    color = Color.PRIMARY,
+}) => {
+    if (!isActive) {
+        return null;
+    }
+    return (
```

Comment:
I think better approach is:
`return ({isActive ? <View
            style={[
                globalStyles.width100,
                globalStyles.height100,
                globalStyles.justifyContentCenter,
                styles.wrapper,
            ]}
        >
            <ActivityIndicator color={color} size="large" />
        </View> : null})`

Reply:
@likeri29 syntax like this more readable, as for me
so I think it's best to stick with it!

@mskmee I am with you 👍